includes/DomainCheckJob.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Miraheze\RequestSSL;
+
+use GenericParameterJob;
+use Job;
+use MediaWiki\MediaWikiServices;
+use User;
+
+class DomainCheckJob extends Job implements GenericParameterJob {
+	public function __construct( array $params ) {
+		parent::__construct( 'DomainCheckJob', $params );
+	}
+	public function run() {
+		$mwServices = MediaWikiServices::getInstance();
```

Comment:
We can use JobSpecification to do service injection, see how I did ImportDumpJob in miraheze/ImportDump@fad211c

Reply:
Done in [331885a](https://github.com/miraheze/RequestSSL/pull/40/commits/331885ac4f8098e004c706c4ef284652fe456e81)

Reply:
JobSpecification pending.